From 0b406b50191a2e6930a09e0f1c4533933be31a05 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 21 Feb 2014 15:57:23 -0500
Subject: [PATCH 2/4] Support parallel-installed Django eggs

If a system has different versions of Django installed as separate
eggs (such as having the latest version as well as an older version
for compatibility with ReviewBoard), this addition to the WSGI
loader enables it to find the matching supported version on the
system.

This will also make things easier on distro upgrades, as it will be
possible to have both the Django 1.4 egg on the system with Review
Board 1.7.21 and Django 1.6 in place for Review Board 2.0
---
 reviewboard/cmdline/conf/reviewboard.wsgi.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/reviewboard/cmdline/conf/reviewboard.wsgi.in b/reviewboard/cmdline/conf/reviewboard.wsgi.in
index cd4b1b888c9b94e20298c035613d3c746bfb41eb..fc3d279ec1b114d1ed07d1f2a4d86ecec74e91a3 100644
--- a/reviewboard/cmdline/conf/reviewboard.wsgi.in
+++ b/reviewboard/cmdline/conf/reviewboard.wsgi.in
@@ -1,3 +1,7 @@
+import __main__
+__main__.__requires__=['Reviewboard']
+import pkg_resources
+
 import os
 import sys
 
-- 
1.8.5.3

